test/math/FloatComplexVectorListTest.ooc

```diff
@@ -19,39 +19,39 @@ FloatComplexVectorListTest: class extends Fixture {
 	complexNumber3 := FloatComplex new (-2, -1)
 
 	complexNumberArray := FloatComplexVectorList new(4)
-	complexNumberArray add(complexNumber0)
-	complexNumberArray add(complexNumber1)
-	complexNumberArray add(complexNumber2)
-	complexNumberArray add(complexNumber3)
+	this complexNumberArray add(this complexNumber0)
+	this complexNumberArray add(this complexNumber1)
+	this complexNumberArray add(this complexNumber2)
+	this complexNumberArray add(this complexNumber3)
```

Comment:
Should void expressions with order dependent side effects be allowed in the declaration section?